src/connectors/asana/functions/create_task.ts

```diff
@@ -9,7 +9,6 @@ export default DefineConnector({
     properties: {
       workspace_gid: { type: Schema.types.string, title: "Workspace" },
       project: { type: Schema.types.string, title: "Project" },
-      assignee_section: { type: Schema.types.string, title: "Section" },
```

Comment:
I believe this is a breaking change too; if a workflow provides the `assignee_section` to this function, what would happen?

Reply:
Yess this may cause some breaking changes